roles/chart_verifier/tasks/tests.yml

```diff
@@ -34,35 +43,36 @@
       when:
         - file_response.status | int != 200
 
-    - name: "Check if the chart values file exists"
+    - name: "Download chart file"
+      get_url:
+        url: "{{ chart.chart_file }}"
+        dest: "{{ work_chart_dir }}/"
+        mode: "0750"
```

Comment:
Don't think we need it +w, can we do 0640?

Reply:
done